sqlglot/dialects/singlestore.py

```diff
@@ -353,6 +353,9 @@ class Generator(MySQL.Generator):
             exp.Variance: rename_func("VAR_SAMP"),
             exp.VariancePop: rename_func("VAR_POP"),
             exp.Xor: bool_xor_sql,
+            exp.Cbrt: lambda self, e: self.sql(
+                exp.Pow(this=e.this, expression=exp.Literal.number(1 / 3))
+            ),
```

Comment:
Should this be a `1 / 3` expression instead of evaluating it?

Reply:
@AdalbertMemSQL I meant generating a literal `1 / 3`, not replacing with `0.3`. Basically, push down the `1 / 3` to the engine.

Reply:
Got it